qiskit/circuit/library/standard_gates/x.py

```diff
@@ -1116,21 +1202,40 @@ def __new__(
         dirty_ancillas: bool = False,  # pylint: disable=unused-argument
         label: Optional[str] = None,
         ctrl_state: Optional[Union[str, int]] = None,
+        *,
+        duration=None,
+        unit="dt",
+        _base_label=None,
     ):
         """Create a new MCX instance.
 
         This must be defined anew to include the additional argument ``dirty_ancillas``.
         """
-        return super().__new__(cls, num_ctrl_qubits, label=label, ctrl_state=ctrl_state)
+        return super().__new__(
+            cls,
+            num_ctrl_qubits,
+            label=label,
+            ctrl_state=ctrl_state,
+            _base_label=_base_label,
+            duration=duration,
+            unit=unit,
+        )
 
     def __init__(
         self,
         num_ctrl_qubits: int,
         dirty_ancillas: bool = False,
         label: Optional[str] = None,
         ctrl_state: Optional[Union[str, int]] = None,
+        _base_label=None,
```

Comment:
Does this need to have `unit` and `duration` as well?

Reply:
This one gets tricky based on the how `MCXGate` works. For some gates it calls `__init__` from `__new__` of `MCXGate()` and tracking the args gets confusing because of the layer of indirection. I'll double check these really quickly to see if they're missing something.